packages/sync-actions/src/prices-actions.js

```diff
@@ -1,12 +1,10 @@
 /* eslint-disable max-len */
-import {
-  buildBaseAttributesActions,
-} from './utils/common-actions'
-
+import { buildBaseAttributesActions } from './utils/common-actions'
 
 export const baseActionsList = [
   { action: 'changeValue', key: 'value' },
   { action: 'setDiscountedPrice', key: 'discounted' },
+  { action: 'setPriceTiers', key: 'tiers' },
```

Comment:
Thats pretty easy to follow 👍 , and it will do the job, maybe also add todo to add the other two actions `addPriceTier`, `removePriceTier` which will make it more accurate, wdyt?